src/kOS/AddOns/ActionGroupsExtended/ActionGroupsExtendedAPI.cs

```diff
@@ -26,6 +26,10 @@ public ActionGroupsExtendedAPI()
         /// <returns>Is AGX installed?</returns>
         public bool Installed() 
         {
+            if (calledType == null)
+            {
+                return false;
+            }
             try //try-catch is required as the below code returns a NullRef if AGX is not present.
```

Comment:
I wasn't sure if there was still a risk for exception on the reflection call to `InvokeMember`.  I suppose that's unlikely to be a null reference, but I still like wrapping this call with `try` so that if the member name ever changes we don't pass the exception upstream.  In our other reflection API's we usually store the member's info itself, which ensures that we know the member/method exists.

Net result: yeah, we should edit the comment so it isn't so specific, but I'd like to keep the `try`.